plugin/src/cli_utils/upgrade/helm_values.rs

```diff
@@ -43,6 +43,26 @@ struct EngineEnabled {
     enabled: bool,
 }
 
+#[derive(Deserialize, Debug)]
+struct HelmConfigValues {
+    engines: Option<HelmConfigValuesEngines>,
+}
+
+#[derive(Deserialize, Debug)]
+struct HelmConfigValuesEngines {
+    replicated: Option<HelmConfigValuesEnginesReplicated>,
+}
+
+#[derive(Deserialize, Debug)]
+struct HelmConfigValuesEnginesReplicated {
+    mayastor: Option<HelmConfigEngineEnabled>,
```

Comment:
Why no directly Option<bool> here? As the HelmConfigEngineEnabled is specifically talking about the enablement so I don't think it will have more fields in future?

Reply:
The whole thing may be missing from the secret. These things show up only if someone adds configuration explicitly.

Reply:
Also, there's no guarantee we'll not have more fields in the future, there's no reason to make this assumption?